contracts/sg721-imago/src/state.rs

```diff
@@ -0,0 +1,23 @@
+use cosmwasm_std::{Addr, Decimal};
+use cw_storage_plus::{Item};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct CollectionInfo<T> {
+    pub creator: String,
+    pub description: String,
+    pub image: String,
+    pub external_link: Option<String>,
+    pub royalty_info: Option<T>,
+}
```

Comment:
we added a new StartTradingTime in upstream, not sure if it will make sense for this since it's basically mint on demand and can take longer to fully mint a project, but if you really want to specify a start trade time in the future should consider adding it .

Reply:
what does `start_trading_time` do?

Reply:
the other way around, if there is no start trading time it will mean that is available for trading right away.

Reply:
Ok I see. That's ok with me to allow trading right away.